e2e/test/scenarios/onboarding/home/browse.cy.spec.ts

```diff
@@ -36,6 +36,24 @@ describeWithSnowplow("scenarios > browse", () => {
     });
   });
 
+  it("can browse to a model in a new tab by meta-clicking", () => {
+    cy.on("window:before:load", win => {
+      // prevent Cypress opening in a new window/tab and spy on this method
+      cy.stub(win, "open").as("open");
+    });
+    cy.visit("/browse/models");
+    cy.findByRole("heading", { name: "Orders Model" }).click({
+      metaKey: true,
```

Comment:
This should depend on the platform, see https://github.com/metabase/metabase/blob/92d615073a3ba31710e26be9f2e83141352d1d1c/e2e/test/scenarios/question/notebook-link-to-data-source.cy.spec.ts#L42-L48